src/main/java/com/wavefront/topology/TraceType.java

```diff
@@ -0,0 +1,23 @@
+package com.wavefront.topology;
+
+import com.wavefront.datastructures.Distribution;
+import com.wavefront.datastructures.ErrorCondition;
+
+import java.util.List;
+import java.util.Random;
+
+/**
+ * A full-signature traceType.
+ *
+ * @author Davit Baghdasaryan (dbagdasarya@vmware.com)
+ */
+public class TraceType {
```

Comment:
also it makes sense to make fields private and use lombok for getters and setters

Reply:
I will keep in mind this for future work, but this is a stupid data structure without any functionality. If you don't mind I prefer to leave it with public modifiers.